src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Migration/v0.01/02-setup-procedures.sql

```diff
@@ -0,0 +1,35 @@
+-- Function: get_all_current_changes
+CREATE OR REPLACE FUNCTION delegation.get_all_current_changes(
+	_altinnappids character varying[],
+	_offeredbypartyids integer[],
+	_coveredbypartyids integer[],
+	_coveredbyuserids integer[])
+    RETURNS SETOF delegation.delegationchanges 
+    LANGUAGE 'sql'
+AS $BODY$
+SELECT
+delegationchangeid,
+altinnappid,
+offeredbypartyid,
+coveredbypartyid,
+coveredbyuserid,
+performedbyuserid,
+blobstoragepolicypath,
+blobstorageversionid,
+isdeleted,
+created
+FROM delegation.delegationchanges
+	INNER JOIN
+	(
+		SELECT MAX(delegationChangeId) AS maxChange
+	 	FROM delegation.delegationchanges
+		WHERE
+		  isdeleted = False
```

Comment:
'isdeleted = False' 
won't work here, as that will exclude the last change when the policy has been deleted and in stead find the second last change event in stead.

so isdeleted changes must be included in result but discarded in code for decision evaluation

Reply:
Fixed and added unittest for the scenario